build.gradle

```diff
@@ -80,6 +80,13 @@ test {
     }
 }
 
+repositories { mavenCentral() }
```

Comment:
@MarcAntoine-Arnaud repositories( ) has already been defined at an earlier location in build.gradle. Is there a need to re-define it?